bumblebee/BumblebeeSource.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
```

Comment:
You would never need to have a header for a source file containing a main function.
Why? Since if another file will include this header you will have a multiple redefinition error (due to the main function).
If you're still not sure, feel free to discuss this with me in private.